testing/integration-tests/build.rs

```diff
@@ -0,0 +1,9 @@
+use cfg_aliases::cfg_aliases;
+
+fn main() {
+    // Setup cfg aliases
+    cfg_aliases! {
+        lightclient: { any(feature = "unstable-light-client", feature = "unstable-light-client-long-running") },
+        fullclient: { all(not(feature = "unstable-light-client"), not(feature = "unstable-light-client-long-running")) },
```

Comment:
dq: Why are these needed?

Reply:
these are just way an easy way to create an alias for `#[cfg(fullclient)]` and `#[cfg(lightclient)` such that we don't have repeat that anymore in the code.

such as:

```rust
// test only for the lightclient
#[cfg(lightclient)]
#[test] 
fn foo () {}
```

Reply:
It allows us to say `cfg(lightclient)` instead of `cfg(any(feature = "unstable-light-client", feature = "unstable-light-client-long-running"))`, to not repeat the longer format